System Analysis/system_analysis.py

```diff
@@ -13,15 +13,45 @@ def word_level(model_name, type):
     analysis = []
 
     # CSV containing Hypothesis and ground truth sentences.
-    Sentdata = pd.read_csv("Eval/" + model_name + "_ALL_DECODING.csv")
+    try:
+        Sentdata = pd.read_csv("Eval/" + model_name + "_ALL_DECODING.csv")
+    except:
+        print(
+            "Unable to read "
+            + model_name
+            + "_ALL_DECODING.csv Check if it is available in Eval folder"
+        )
+        print("skipping analysis...")
+        return
+
     Sentdata = Sentdata.replace(np.nan, "")
     # Checking Data
     # print(Sentdata.info())
 
-    # Ground Truth sentences.
-    ground = list(Sentdata["Actual_" + model_name].values)
-    # Hypothesis sentences for a particular decoding technique.
-    pred = list(Sentdata["Hypothesis_" + type + model_name].values)
+    try:
+        # Ground Truth sentences.
+        ground = list(Sentdata["Actual_" + model_name].values)
+    except:
+        print(
+            "CSV does not have Actual_"
+            + model_name
+            + " attribute. please provide correct model_name"
+        )
+        print("skipping analysis...")
+        return
+
+    try:
+        # Hypothesis sentences for a particular decoding technique.
+        pred = list(Sentdata["Hypothesis_" + type + model_name].values)
+    except:
+        print(
+            "CSV does not have Hypothesis_"
+            + type
+            + model_name
+            + " attribute. please provide correct type and/or model_name"
```

Comment:
attribute`s`. Please provide correct type and/or model_name and check the name format of the .csv column